src/main/thrift/parquet.thrift

```diff
@@ -232,6 +232,7 @@ struct MapType {}     // see LogicalTypes.md
 struct ListType {}    // see LogicalTypes.md
 struct EnumType {}    // allowed for BINARY, must be encoded with UTF-8
 struct DateType {}    // allowed for INT32
+struct Float16Type {} // allowed for FIXED[2], must encoded raw FLOAT16 bytes
```

Comment:
why not allow bit splitting?

Reply:
@emkornfield What do you mean here?

Reply:
Ah, perhaps you mean the BYTE_STREAM_SPLIT encoding?

Reply:
yes. BYTE_STREAM_SPLIT

Reply:
Well, I guess it wouldn't cost much to allow it (implementations would not support it at the start anyway).